compose/sxp/10.4/ltsc2022/xm1/.env

```diff
@@ -0,0 +1,29 @@
+COMPOSE_PROJECT_NAME=sitecore-xm1
+SITECORE_DOCKER_REGISTRY=scr.sitecore.com/sxp/
+SITECORE_VERSION=10.4-ltsc2022
+EXTERNAL_IMAGE_TAG_SUFFIX=ltsc2022
+SITECORE_ADMIN_PASSWORD=
+SQL_SERVER=mssql
+SQL_SA_LOGIN=sa
+SQL_SA_PASSWORD=
+SQL_DATABASE_PREFIX=Sitecore
+SQL_CUSTOM_DATABASE_PREFIX_UPDATE_FROM=
+TELERIK_ENCRYPTION_KEY=
+SITECORE_GRAPHQL_ENABLED=true
+SITECORE_GRAPHQL_EXPOSEPLAYGROUND=false
+# You should change the encryption key to a random string and not use the default value
+SITECORE_GRAPHQL_UPLOADMEDIAOPTIONS_ENCRYPTIONKEY=432A462D4A614E64
+SITECORE_IDSECRET=
+SITECORE_ID_CERTIFICATE=
+SITECORE_ID_CERTIFICATE_PASSWORD=
+SITECORE_LICENSE=
+CD_HOST=xm1cd.localhost
+CM_HOST=xm1cm.localhost
+ID_HOST=xm1id.localhost
+TRAEFIK_IMAGE=traefik:v2.9.8-windowsservercore-1809
```

Comment:
Please consider using the latest version of v2 so that all containers can run in "process" isolation in local dev. For example: **traefik:2.11.2-windowsservercore-ltsc2022**

I have tested **traefik:2.11.0-windowsservercore-ltsc2022** already with 10.2 and 10.3 with "process" isolation on Windows 11. It just works, just change the image tag and isolation - no other changes are needed.

Reply:
@peternazarov latest v3 version of traefik is being tested with all SXP versions at the moment, we'll soon update the public configuration accordingly. Expect the update. :)